### PR TITLE
Dor 48 crear formulario de creacion

### PR DESCRIPTION
**Pull request by AdpSoftware-tech**

### CODE REVIEW

File: src/pages/auth/Login.jsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import userIcon from '../../assets/user.svg';
 import passwordIcon from '../../assets/password.svg';
 import '../../styles/login.css';
 import { useState } from 'react';
+// import BootstrapButton from '../../components/BtnBlue2.jsx';
```
Comment (wmartzh) [inline src/pages/auth/Login.jsx]: Please delete this comments

File: src/pages/auth/Login.jsx [hunk 2]
```diff
@@ -3,6 +3,8 @@ import userIcon from '../../assets/user.svg';
...
+// import axios from 'axios';
 import BootstrapButton from '../../components/btnBlue.jsx';
 // axios.defaults.headers.post['Content-Type'] = 'application/x-www-form-urlencoded';
```
Comment (wmartzh) [inline src/pages/auth/Login.jsx]: Please delete this comments

File: src/pages/students/RegisterStudents.jsx [hunk 3]
```diff
@@ -1,7 +1,125 @@
...
+  });
+
+  const handleInputChange = (event) => {
+    const { name, value } = event.target;
+    setPostStudent((prevFormData) => ({
+      ...prevFormData,
+      [name]: value
+    }));
+  };
+  const token =
```
Comment (wmartzh) [inline src/pages/students/RegisterStudents.jsx]: This code is not required

File: src/pages/students/RegisterStudents.jsx [hunk 4]
```diff
@@ -1,7 +1,125 @@
...
+    '[eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImFkb25pc0Bob21lLmNvbSIsImlhdCI6MTY3ODU5NjY1MCwiZXhwIjoxNjgwNjcwMjUwfQ.b_nI_So4SY3lALiHHfVgKD-U05WMozu1yU_11tUBDzk]';
+
+  const config = {
+    headers: { Authorization: `Bearer ${token}` }
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    try {
+      const response = await axios.post(
```
Comment (wmartzh) [inline src/pages/students/RegisterStudents.jsx]: take this request out of the component

File: src/pages/students/StudentsOverview.jsx [hunk 5]
```diff
@@ -1,7 +1,141 @@
...
+
 const StudentsOverview = () => {
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(5);
+  const [data, setData] = useState([]);
+  const [info, setInfo] = useState({});
+
+  const token = localStorage.getItem('token');
+  useEffect(() => {
+    axios
```
Comment (wmartzh) [inline src/pages/students/StudentsOverview.jsx]: Take this request out the component

File: src/pages/students/StudentsOverview.jsx [hunk 6]
```diff
@@ -1,7 +1,141 @@
...
+                  <TableRow>
+                    {columns.map((column) => (
+                      <TableCell key={column.id} align={column.align}>
+                        {column.label}
+                      </TableCell>
+                    ))}
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {data.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((row) => (
```
Comment (wmartzh) [inline src/pages/students/StudentsOverview.jsx]: Backend already do the pagination, so we have to manage this values on the request

File: src/pages/auth/Login.jsx (hunk 1 above)
Comment (AdpSoftware-tech) [reply src/pages/auth/Login.jsx]: deleted comments

File: src/pages/auth/Login.jsx (hunk 2 above)
Comment (AdpSoftware-tech) [reply src/pages/auth/Login.jsx]: deleted comments

File: src/pages/students/RegisterStudents.jsx (hunk 3 above)
Comment (AdpSoftware-tech) [reply src/pages/students/RegisterStudents.jsx]: console.log deleted

File: src/pages/students/RegisterStudents.jsx (hunk 4 above)
Comment (AdpSoftware-tech) [reply src/pages/students/RegisterStudents.jsx]: teacher is not ready yet, they are tests that I did but it didn't work I don't know why

File: src/pages/students/StudentsOverview.jsx (hunk 6 above)
Comment (AdpSoftware-tech) [reply src/pages/students/StudentsOverview.jsx]: ok teacher

Comment (AdpSoftware-tech) [reply src/pages/students/StudentsOverview.jsx]: listo profe

File: src/pages/students/StudentsOverview.jsx (hunk 5 above)
Comment (AdpSoftware-tech) [reply src/pages/students/StudentsOverview.jsx]: Listo

File: src/pages/students/StudentsOverview.jsx [hunk 7]
```diff
@@ -30,8 +30,8 @@ const StudentsOverview = () => {
   const [rowsPerPage, setRowsPerPage] = useState(5);
   const [data, setData] = useState([]);
 
-  const token = localStorage.getItem('token');
   useEffect(() => {
+    const token = localStorage.getItem('token');
```
Comment (wmartzh) [inline src/pages/students/StudentsOverview.jsx]: This should be deleted

**Review by wmartzh — CHANGES_REQUESTED** [review]

File: src/pages/students/StudentsOverview.jsx (hunk 5 above)
Comment (AdpSoftware-tech) [reply src/pages/students/StudentsOverview.jsx]: listo he sacado la solicitud del componente

File: src/pages/students/RegisterStudents.jsx [hunk 8]
```diff
@@ -1,7 +1,120 @@
...
+    const { name, value } = event.target;
+    setPostStudent((prevFormData) => ({
+      ...prevFormData,
+      [name]: value
+    }));
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const token = localStorage.getItem('token');
```
Comment (wmartzh) [inline src/pages/students/RegisterStudents.jsx:44]: This should be removed

File: src/pages/students/RegisterStudents.jsx [hunk 9]
```diff
@@ -1,7 +1,120 @@
...
+    try {
+      const response = await axios.post('http://localhost:8000/student/register', postStudent, {
```
Comment (wmartzh) [inline src/pages/students/RegisterStudents.jsx:48]: This axios call should be moved to external file like a service

**Review by wmartzh — CHANGES_REQUESTED** [review]
All axios request should be moved to external files

File: src/pages/students/RegisterStudents.jsx (hunk 9 above)
Comment (AdpSoftware-tech) [reply src/pages/students/RegisterStudents.jsx:48]: profe esto aun no esta listo no funciona aun

File: src/pages/students/StudentsOverview.jsx (hunk 7 above)
Comment (AdpSoftware-tech) [reply src/pages/students/StudentsOverview.jsx]: listo

**Pull request merged**